lib/services/nsStarterKitsNpmService.ts

```diff
@@ -7,7 +7,7 @@ const defaultHeaders = {
 export class NsStarterKitsNpmService implements INsStarterKitsNpmService {
     installPageTemplate(pageName: string, flavor: string, templatesDirectory: string): Promise<any> {
         return new Promise((resolve, reject) => {
-            const command: string = (process.platform.indexOf("win") > -1) ? "npm.cmd" : "npm";
+            const command: string = (process.platform.startsWith("win")) ? "npm.cmd" : "npm";
```

Comment:
why not directly checking if `process.platform === "win32"`

Reply:
https://nodejs.org/api/process.html#process_process_platform on Windows the value is always `win32`

Reply:
I'm ok for both 'win32' or 'win'  
same for 64 bit